examples/reactnative/android/app/src/main/java/com/reactnative/MainApplication.java

```diff
@@ -46,7 +46,6 @@ public ReactNativeHost getReactNativeHost() {
   @Override
   public void onCreate() {
     super.onCreate();
-    BugsnagReactNativePlugin.register();
```

Comment:
I'm surprised to see this removed and not see the plugin added to configuration. How does the plugin now load itself?